sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/PushDownUtils.scala

```diff
@@ -130,8 +128,17 @@ object PushDownUtils extends PredicateHelper {
       return None
     }
 
-    val agg = new Aggregation(translatedAggregates.toArray, translatedGroupBys.toArray)
-    Some(agg).filter(scanBuilder.pushAggregation)
+    Some(new Aggregation(translatedAggregates.toArray, translatedGroupBys.toArray))
+  }
+
+  /**
+   * Pushes down aggregates to the data source reader
+   *
+   * @return pushed aggregation.
+   */
+  def pushAggregates(
+      scanBuilder: SupportsPushDownAggregates, aggOpt: Option[Aggregation]): Option[Aggregation] = {
+    aggOpt.filter(scanBuilder.pushAggregation)
```

Comment:
Doesn't seem necessary to create a method for only one line of code.

Reply:
Yeah. I'm a little uncertain, so I'm waiting for your comment.